<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtVTKRenderWindows</class>
 <widget class="QMainWindow" name="QtVTKRenderWindows">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1243</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtVTKRenderWindows</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/Icons/help.png</normaloff>:/Icons/help.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>22</width>
    <height>22</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>611</width>
      <height>561</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QVTKOpenGLWidget" name="imageWindow" native="true"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="toggleCandidatesButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>580</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Candidates</string>
    </property>
   </widget>
   <widget class="QPushButton" name="toggleModelButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>580</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Model</string>
    </property>
   </widget>
   <widget class="QPushButton" name="togglePlanesButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>580</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Image Plane</string>
    </property>
   </widget>
   <widget class="QLabel" name="frameLabel">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>580</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Frame Number</string>
    </property>
   </widget>
   <widget class="QSlider" name="frameSlider">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>580</y>
      <width>401</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="frameValue">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>580</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>00</string>
    </property>
   </widget>
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>601</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>601</width>
       <height>419</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Boundary Candidates</string>
     </attribute>
     <widget class="QPushButton" name="calculateCandidatesButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>151</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Calculate Candidates</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>601</width>
       <height>419</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Orientation</string>
     </attribute>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>601</width>
        <height>41</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="edButton">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>10</y>
         <width>111</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Capture</string>
       </property>
      </widget>
      <widget class="QLabel" name="edValue">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>10</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Value</string>
       </property>
      </widget>
      <widget class="QLabel" name="edLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>81</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>ED Frame:</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>601</width>
       <height>419</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Model</string>
     </attribute>
     <widget class="QPushButton" name="registerButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>121</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="cellDataGroup">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>131</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Cell Coloring</string>
      </property>
      <widget class="QRadioButton" name="cellDataButtonNone">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>116</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>None</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="cellDataButtonSQUEEZ">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>116</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>SQUEEZ</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="cellDataButtonResidual">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>116</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Residual</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="toggleWiresButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>250</y>
        <width>101</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Model Wires</string>
      </property>
     </widget>
     <widget class="QPushButton" name="toggleColorbarButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>280</y>
        <width>81</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Colorbar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="toggleSurfaceButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>220</y>
        <width>111</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Model Surface</string>
      </property>
     </widget>
     <widget class="QPushButton" name="toggleResidualsButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>310</y>
        <width>131</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Model Residuals</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="frameIncrement">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>580</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="frameDecrement">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>580</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="writeScreenshots">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Screenshots</string>
    </property>
   </widget>
   <widget class="QPushButton" name="jumpToEDButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>540</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>End Diastole</string>
    </property>
   </widget>
   <widget class="QLabel" name="frameLabel_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>540</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Jump to Frame:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLWidget</class>
   <extends>QWidget</extends>
   <header>QVTKOpenGLWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
